.github/workflows/auto-labeler.yml

```diff
@@ -1,24 +1,24 @@
 ---
-  name: Auto Labeler
+name: Auto Labeler
 
-  on:
-    # pull_request_target event is required for autolabeler to support all PRs including forks
-    pull_request_target:
-      types: [opened, reopened, synchronize]
+on:
+  # pull_request_target event is required for autolabeler to support all PRs including forks
+  pull_request_target:
+    types: [ opened, reopened, edited, synchronize ]
```

Comment:
meat of change, adding `edited` here

Reply:
agreed. like a light meat. A single slice of pepperoni. 👨🏻‍🍳